containers/singlesite/wp-config.php

```diff
@@ -84,7 +84,8 @@
  *
  * @link https://codex.wordpress.org/Debugging_in_WordPress
  */
-define('WP_DEBUG', false);
+define('WP_DEBUG', true);
+define('WP_DEBUG_LOG', true);
```

Comment:
If the purpose of this is to pipe debug info to the log file instead of displaying it in the UI, should we add `define('WP_DEBUG_DISPLAY', false);` as well?